pkg/kds/mux/session.go

```diff
@@ -28,21 +33,32 @@ type MultiplexStream interface {
 	Context() context.Context
 }
 
-func NewSession(peerID string, stream MultiplexStream) Session {
+// NewSession creates a multiplexed session for KDS so both CP sends and receives resources.
+//
+// Buffer settings recommendations:
+// The buffer size should be of a size of all inflight request, so we are never blocked on the buffer channels.
```

Comment:
```suggestion
// The buffer size should be of a size of all inflight request, so we are never block on the buffer channels.
```

Reply:
did you mean `so we never block`? (without `are`)

Reply:
Can't read nevermind!